app/templates/docs/action.html

```diff
@@ -1,615 +1,643 @@
+                  <li><code class="docutils literal"><span class="pre">dateLastActivity</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">desc</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">descData</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">due</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">email</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idAttachmentCover</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idBoard</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idChecklists</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idLabels</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idList</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idMembers</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idMembersVoted</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idShort</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">labels</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">manualCoverAttachment</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">name</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">pos</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">shortLink</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">shortUrl</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">subscribed</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">url</span></code></li>
+                </ul>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </div>
+  <div class="section" id="get-1-actions-idaction-display">
+    <h2>GET <span class="target" id="actions-idaction-display">/1/actions/[idAction]/display</span><a class="headerlink" href="#get-1-actions-idaction-display" title="Permalink to this headline"></a></h2>
+    <ul class="simple">
+      <li><strong>Required permissions:</strong> read</li>
+      <li><strong>Arguments:</strong> None</li>
+    </ul>
+  </div>
+  <div class="section" id="get-1-actions-idaction-entities">
+    <h2>GET <span class="target" id="actions-idaction-entities">/1/actions/[idAction]/entities</span><a class="headerlink" href="#get-1-actions-idaction-entities" title="Permalink to this headline"></a></h2>
```

Comment:
An example is definitely useful here:

`GET /1/actions/560bf78ea40a4590ca3c5c7d/entities`

```json
[
    {
        "type": "member",
        "id": "556c8537a1928ba745504dd8",
        "username": "mattcowan",
        "text": "Matt Cowan"
    },
    {
        "type": "text",
        "text": "renamed"
    },
    {
        "type": "card",
        "id": "560bf587882bb89c3882294e",
        "shortLink": "1UtQeVnQ",
        "text": "Mount Rainier National Park"
    },
    {
        "type": "text",
        "text": "(from Mount Rainier National Park | MapQuest National Parks)"
    }
]
```